apps/src/main.c

```diff
@@ -1,60 +1,63 @@
-#include "stdio.h"
-#include "tcp_context.h"
 #include <signal.h>
 #include <string.h>
 
+#include "client.h"
+#include "stdio.h"
+#include "tcp_context.h"
+
 static char buffer[80];
 
-static void callback(const event_type ev, void *c_info, const void *in,
+static void callback(const event_type ev, void* client, const void* in,
                      const unsigned int len) {
-  const char *client_ip = socev_get_client_ip(c_info);
-  const unsigned short port = socev_get_client_port(c_info);
+  const char* client_ip = client_get_ip(client);
+  const unsigned short port = client_get_port(client);
```

Comment:
can be use uint16_t instead of unsigned short

Reply:
done